src/build.ts

```diff
@@ -0,0 +1,23 @@
+
+// expected injected enviornment variables
+declare const GTM: any;
+declare const SENTRY: any;
+declare const PROD_MODE: any;
+
+interface BuildConfig {
+  enableGoogleTagManager: boolean;
+  enableSentry: boolean;
+  PROD_MODE: boolean;
+}
+
+const BUILD: BuildConfig = {
+  enableGoogleTagManager: Boolean(parse(GTM)),
+  enableSentry: Boolean(parse(SENTRY)),
+  PROD_MODE: Boolean(parse(PROD_MODE)),
+};
+
+function parse(stringified) {
+  return JSON.parse(stringified || null);
+}
+
+export default BUILD;
```

Comment:
I'm not a huge fan of default exports. Can we make this a named export. Also can we call it `buildConfig` or something.